### PR TITLE
Past event access

### PR DESCRIPTION
**Pull request by pcantrell**
Adds the ability to view past events via`/events/___` and `/events/___/sessions` routes. The existing `/home` and `/sessions` routes continue to work, and continue to show the current event. The new routes also support the special ID `current`.

Session listings and individual events from past MinneBars clearly indicate that they are from old events. Session voting and editing are hidden from the UI (though not prohibited by any server-side security rules) for past events. The UI could use some further love to help people discover and navigate events — there is, for example, no events index page — but this should at least serve the immediate need of giving presenters access to their defunct spring 2020 session descriptions even while the fall 2020 event is active.

### CODE REVIEW

File: src/app/controllers/events_controller.rb [hunk 1]
```diff
@@ -1,8 +1,19 @@
 class EventsController < ApplicationController
   def show
-    @event = Event.includes(:sessions).find(params[:id])
+    @event = if params[:id] == 'current'
+      Event.current_event
```
Comment (aackerman) [inline src/app/controllers/events_controller.rb:4]: Does this also need `.includes(:sessions)`? I'm not really sure.

Comment (pcantrell) [reply src/app/controllers/events_controller.rb:4]: IIRC there's a huge SQL query storm if it's not there. Could try making the request and checking the logs to be sure….

File: src/config/routes.rb [hunk 2]
```diff
@@ -1,8 +1,14 @@
 Sessionizer::Application.routes.draw do
   root to: 'schedules#index'
 
-  get '/home' => 'pages#home', as: :home_page
+  get '/home' => 'events#show', id: 'current', as: :home_page
 
+  # Session actions available for all events
+  resources :events do
```
Comment (aackerman) [inline src/config/routes.rb:7]: Do we still need `resources :events, only: [:show] do`?

Comment (pcantrell) [reply src/config/routes.rb:7]: I think you're right: we're only using `show` right now. And probably will only want to add `index` in the future.

File: src/app/controllers/events_controller.rb (hunk 1 above)
Comment (pcantrell) [reply src/app/controllers/events_controller.rb:4]: This was a good catch: it turns out _neither_ branch needs `.includes(:sessions)`.

(The JSON form is a hot query mess, but I'm not sure anybody’s actually using it, so I left the mess hot.)

**Comment by pcantrell** [comment]
Calling it good.

**Pull request merged**